qiskit_finance/applications/estimation/estimation_application.py

```diff
@@ -10,16 +10,20 @@
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
 
-"""The European Call Option Expected Value."""
+"""An abstract class for estimation application classes."""
 from abc import ABC, abstractmethod
 
+from qiskit.algorithms.amplitude_estimators import EstimationProblem
+
 
 class EstimationApplication(ABC):
```

Comment:
Do we expect to have an `interpret` method in here similar to OptimizationApplication

Reply:
Honestly, somehow, I didn't even think about it, but you're right. It's better and natural to have an `interpret` method.